src/components/Home.js

```diff
@@ -0,0 +1,67 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSearch, faWindowClose } from '@fortawesome/fontawesome-free-solid';
+import { useState } from 'react';
+
+import { NavLink } from 'react-router-dom';
+
+const Home = () => {
+  const [sideBar, setSideBar] = useState('none');
+  const [navBar, setNavBar] = useState('current-nav');
+
+  const handleOpen = () => {
+    setSideBar('current-nav');
+    setNavBar('none');
+  };
+
+  const handleClose = () => {
+    setNavBar('current-nav');
+    setSideBar('none');
+  };
+
+  return (
+    <div className="home">
+      <nav className={navBar}>
+        <FontAwesomeIcon icon="fa-solid fa-bars" onClick={handleOpen} />
+        <FontAwesomeIcon
+          icon={faSearch}
+          className="search"
+          onClick={() => alert('Please signup or signin!')}
```

Comment:
- [ ] According to the general [Javascript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) best practices requirements, please kindly don't use any alerts.  

You could use a modal, a simple message, a text that appears, etc. 👍